frontend/src/utils/util.ts

```diff
@@ -68,6 +68,15 @@ export const getSelectedPartFromURL = (urlParts: string[]): string => {
   }
 };
 
+export const capitalizeFirstLetter = (inputString: string): string => {
```

Comment:
If we utilize a CSS class, it would be possible to eliminate the need for this particular function.